### ISSUE TITLE
Suggestion: Add more roles

### ISSUE THREAD

**Issue by HBIDamian** [MinekCZ/PalermoTown#1]
On the PocketMine Discord server, I've posted a plugin suggestion back in late August 2021. 
This plugin seems to be the closest to it now. So I suggest adding more roles.

I called the idea: "TownOfPMMP". Here's what I typed on the thread in discord:
> An idea I think the community would like is a plugin that's based on the game "Town Of Salem". 
> Or for those really "Sus" like users, the Among Us mod called "Town of Us".
> It would make for an interesting gamemode.
> 
> Before the game starts, players are put into a lobby where the host can select what roles will be in the game. Players are then assigned roles at random from the list of chosen roles. Players have an in-game role card that explains their roles abilities and alignments.
> For example: https://github.com/polusgg/Town-Of-Us
> 
> Another great example is https://store.steampowered.com/app/1568590/Goose_Goose_Duck/. In fact, that's literally my whole pitch right there. 😂

**Comment by MinekCZ** [comment]
Yeah, thas is possible, I can make it, but Poggit is still not approving my plugin. So It's all about time...

**Comment by MinekCZ** [comment]
So no motivation to continue
